db/migrate/20210120101859_add_null_fields_to_framework_responses.rb

```diff
@@ -0,0 +1,6 @@
+class AddNullFieldsToFrameworkResponses < ActiveRecord::Migration[6.0]
+  def change
+    change_column :framework_responses, :section, :string, null: false
```

Comment:
would be good to check that there are no existing nulls in `framework_responses.section` and `framework_responses.value_type`, otherwise this migration will fail